### ISSUE TITLE
Enable/disable connection test fails frequently

### ISSUE THREAD

**Issue by majorz** [resin-io-modules/network_manager#43]
The `connection::test_enable_disable_functions` fails frequently with the following error message:

```
---- connection::test_enable_disable_functions stdout ----
	thread 'connection::test_enable_disable_functions' panicked at 'called `Result::unwrap()` on an `Err` value: D-Bus error: No such interface 'org.freedesktop.DBus.Properties' on object at path /org/freedesktop/NetworkManager/ActiveConnection/4 (org.freedesktop.DBus.Error.UnknownMethod)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
```

**Comment by josephroberts** [comment]
Perhaps try adding a small delay [here](https://github.com/resin-io-modules/network_manager/blob/master/src/connection/mod.rs#L184) and [here](https://github.com/resin-io-modules/network_manager/blob/master/src/connection/mod.rs#L191)

**Comment by majorz** [comment]
Adding even big delays there did not work

**Comment by majorz** [comment]
Fixed with PR #44
